.ci/magician/cmd/check_cassettes.go

```diff
@@ -62,7 +62,7 @@ var checkCassettesCmd = &cobra.Command{
 			os.Exit(1)
 		}
 
-		ctlr := source.NewController(env["GOPATH"], "modular-magician", env["GITHUB_TOKEN"], rnr)
+		ctlr := source.NewController(env["GOPATH"], "modular-magician", env["GITHUB_TOKEN_DOWNSTREAMS"], rnr)
```

Comment:
Here & above

```suggestion
		ctlr := source.NewController(env["GOPATH"], "modular-magician", env["GITHUB_TOKEN_CLASSIC"], rnr)
```

Reply:
I think this step can use the fine-grained token since it's only cloning the downstreams.

Reply:
sure, you could pass in GITHUB_TOKEN_DOWNSTREAMS instead if the classic token isn't needed - but what's being passed in needs to match what's being referenced.